app/configurations/config.hsl.js

```diff
@@ -90,6 +90,7 @@ export default {
   useRoutingFeedbackPrompt: true,
 
   feedIds: ['HSL', 'HSLlautta', 'Sipoo'],
+  internalFeedIds: ['HSL'],
 
```

Comment:
It will surely be easier to maintain list of external feeds.

Reply:
This has been changed to externalFeedIds